setup/src/common/scripts/postStartCommand.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Put your "post start" actions here:
```

Comment:
Should we possible remove this line as a "user" cannot just change these files as then the sync will complain that the files have been modified.

Reply:
I guess, you're right. Removed it.